eudi_wallet_python/satosa/backend.py

```diff
@@ -0,0 +1,163 @@
+        return Response(
+            text_type(
+                """
+                {
+                    "request: "ewogICJ0eXAiOiAiZHBvcCtqd3QiLAogICJhbGciOiAiRVMyNTYiLAogICJqd2siOiB7CiAgICAia3R5IjogIkVDIiwKICAgICJ4IjogImw4dEZyaHgtMzR0VjNoUklDUkRZOXpDa0RscEJoRjQyVVFVZldWQVdCRnMiLAogICAgInkiOiAiOVZFNGpmX09rX282NHpiVFRsY3VOSmFqSG10NnY5VERWclUwQ2R2R1JEQSIsCiAgICAiY3J2IjogIlAtMjU2IgogIH0KfQ.ewogICJqdGkiOiAiZjQ3Yzk2YTEtZjkyOC00NzY4LWFhMzAtZWYzMmRjNzhhYTY5IiwKICAiaHRtIjogIkdFVCIsCiAgImh0dSI6ICJodHRwczovL3ZlcmlmaWVyLmV4YW1wbGUub3JnL3JlcXVlc3RfdXJpIiwKICAiaWF0IjogMTU2MjI2MjYxNiwKICAiYXRoIjogImZVSHlPMnIyWjNEWjUzRXNOcldCYjB4V1hvYU55NTlJaUtDQXFrc21RRW8iCn0"
+                }
+                """
+            ).encode("utf-8"), content="text/json; charset=utf8"
+        )
+    
+    def request_enpoint(self, context, *args):
+        return Response(
+            text_type(
+                """
+                {
+                    "response: "ewogICJhbGciOiAiRVMyNTYiLAogICJ0eXAiOiAiSldUIiwKICAia2lkIjogImUwYmJmMmYxLThjM2EtNGVhYi1hOGFjLTJlOGYzNGRiOGE0NyIKfQ.ewogICJpc3MiOiAiaHR0cHM6Ly93YWxsZXQtcHJvdmlkZXIuZXhhbXBsZS5vcmcvaW5zdGFuY2UvdmJlWEprc000NXhwaHRBTm5DaUc2bUN5dVU0amZHTnpvcEd1S3ZvZ2c5YyIsCiAgImp0aSI6ICIzOTc4MzQ0Zi04NTk2LTRjM2EtYTk3OC04ZmNhYmEzOTAzYzUiLAogICJhdWQiOiAiaHR0cHM6Ly92ZXJpZmllci5leGFtcGxlLm9yZy9jYWxsYmFjayIsCiAgImlhdCI6IDE1NDE0OTM3MjQsCiAgImV4cCI6IDE1NzMwMjk3MjMsCiAgIm5vbmNlIjogIm4tMFM2X1d6QTJNaiIsCiAgInZwIjogIjxTRC1KV1Q-fjxEaXNjbG9zdXJlIDE-fjxEaXNjbG9zdXJlIDI-fi4uLn48RGlzY2xvc3VyZSBOPiIKfQ"
+                }
+                """
+            ).encode("utf-8"), content="text/json; charset=utf8"
+        )
+
+    def _metadata_endpoint(self, context):
+        """
+        Endpoint for retrieving the backend metadata
+        :type context: satosa.context.Context
+        :rtype: satosa.response.Response
+
+        :param context: The current context
+        :return: response with metadata
+        """
+        logger.debug("Sending metadata response")
+        self.sp.config
+
+        """metadata = entity_descriptor(conf)
+        
+        # metadata signature
+        secc = security_context(conf)
+        #
+        sign_dig_algs = self.get_kwargs_sign_dig_algs()
+        eid, xmldoc = sign_entity_descriptor(
+            metadata, None, secc, **sign_dig_algs)
+
+        valid_instance(eid)"""
+        return Response(
+            text_type("").encode("utf-8"), content="text/xml; charset=utf8"
+        )
+
+    def get_kwargs_sign_dig_algs(self):
+        kwargs = {}
+        # backend support for selectable sign/digest algs
+        alg_dict = dict(signing_algorithm="sign_alg",
+                        digest_algorithm="digest_alg")
+        for alg in alg_dict:
+            selected_alg = self.config["sp_config"]["service"]["sp"].get(alg)
+            if not selected_alg:
+                continue
+            kwargs[alg_dict[alg]] = selected_alg
+        return kwargs
+
+    def check_blacklist(self, context, entity_id):
```

Comment:
this is not of interest, please remove

Reply:
Removed in commit 496786501d97781dfe8c9fceac91b569db06134e